Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/AuditSearchBean.java

```diff
@@ -0,0 +1,269 @@
+            this.auditRecordList.clear();
+        }
+        this.auditMessage = "";
+        this.auditFound = false;
+        return NavigationConstant.AUDIT_SEARCH_PAGE;
+    }
+
+    public String clearAuditTabMessageId() {
+        this.RelatesTo = null;
+        this.messageId = null;
+        if (this.auditRecordList != null && !auditRecordList.isEmpty()) {
+            this.auditRecordList.clear();
+        }
+        this.auditMessage = "";
+        this.auditFound = false;
+        return NavigationConstant.AUDIT_SEARCH_PAGE;
+    }
+
+    private Map<String, String> populateRemoteOrgHcid() {
+
+        return new RemoteOrganizationIdentifier().getRemoteHcidFromUUID();
+
+    }
+
+    private ArrayList<String> populateEventTypeList() {
+        //TODO convert into Map. Define enum and and hold service Names.
+        ArrayList<String> serviceNames = new ArrayList<>();
+        serviceNames.add(NhincConstants.NHINC_XDR_SERVICE_NAME);
+        serviceNames.add(NhincConstants.PATIENT_DISCOVERY_SERVICE_NAME);
+        serviceNames.add(NhincConstants.DOC_RETRIEVE_SERVICE_NAME);
+        serviceNames.add(NhincConstants.DOC_QUERY_SERVICE_NAME);
+        serviceNames.add(NhincConstants.NHIN_ADMIN_DIST_SERVICE_NAME);
+        serviceNames.add(NhincConstants.CORE_X12DS_REALTIME_SERVICE_NAME);
+        serviceNames.add(NhincConstants.CORE_X12DS_GENERICBATCH_REQUEST_SERVICE_NAME);
+        serviceNames.add(NhincConstants.CORE_X12DS_GENERICBATCH_RESPONSE_SERVICE_NAME);
+        serviceNames.add(NhincConstants.PATIENT_DISCOVERY_DEFERRED_REQ_SERVICE_NAME);
+        serviceNames.add(NhincConstants.PATIENT_DISCOVERY_DEFERRED_RESP_SERVICE_NAME);
+        serviceNames.add(NhincConstants.NHINC_XDR_REQUEST_SERVICE_NAME);
+        serviceNames.add(NhincConstants.NHINC_XDR_RESPONSE_SERVICE_NAME);
+        return serviceNames;
+    }
+
+    private Map<String, String> populateEventOutcomeIndicatorValues() {
+        Map<String, String> outcomeList = new HashMap<>();
+        outcomeList.put("Success", "0");
+        outcomeList.put("Failure", "12");
+        return outcomeList;
+    }
+
+    public String getMessageId() {
+        return messageId;
+    }
+
+    public void setMessageId(String messageId) {
+        this.messageId = messageId;
+    }
+
+    public String getRelatesTo() {
+        return RelatesTo;
+    }
+
+    public void setRelatesTo(String RelatesTo) {
+        this.RelatesTo = RelatesTo;
+    }
+
+    public boolean isAuditFound() {
+        return auditFound;
+    }
+
+    public String getAuditMessage() {
+        return auditMessage;
+    }
+
+    public ArrayList<Audit> getAuditRecordList() {
+        return this.auditRecordList;
+    }
+
+    public void getAuditRecordList(ArrayList<Audit> auditRecordList) {
+        this.auditRecordList = auditRecordList;
+    }
+
+    public Map<String, String> getEventOutcomeIndicatorList() {
+        return eventOutcomeIndicatorList;
```

Comment:
We are returning map here. The field and getter name implies that you are returning a list.